cmd/aao/pool.go

```diff
@@ -0,0 +1,56 @@
+package aao
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	//"k8s.io/klog"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+// newCmdPool gets the current status of the AWS Account Operator AccountPool
+func newCmdPool(streams genericclioptions.IOStreams, flags *genericclioptions.ConfigFlags) *cobra.Command {
+	ops := newPoolOptions(streams, flags)
+	poolCmd := &cobra.Command{
+		Use:               "pool",
+		Short:             "Get the status of the AWS Account Operator AccountPool",
+		Args:              cobra.NoArgs,
+		DisableAutoGenTag: true,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmdutil.CheckErr(ops.complete(cmd))
+			cmdutil.CheckErr(ops.run())
+		},
+	}
+
+	return poolCmd
+}
+
+// poolOptions defines the struct for running the pool command
+type poolOptions struct {
+	genericclioptions.IOStreams
+}
+
+func newPoolOptions(streams genericclioptions.IOStreams, _ *genericclioptions.ConfigFlags) *poolOptions {
+	return &poolOptions{
+		IOStreams: streams,
+	}
+}
+
+func (o *poolOptions) complete(cmd *cobra.Command) error {
+	return nil
+}
+
+func (o *poolOptions) run() error {
+	cmd := "oc get accounts -o json -n aws-account-operator | jq '.items | map(select(.status.claimed==\"false\" or .status.claimed==null)) | map(select(.status.state!=\"Failed\")) | map(select(.spec.legalEntity==null or .spec.legalEntity.id==\"\" or .spec.legalEntity.id==null)) | length'"
+	output, err := exec.Command("bash", "-c", cmd).Output()
```

Comment:
Since we are already use the kube client code in osdctl, we can use it to query  accounts from the `aws-account-operator` namespace, and also process the JSON with Go. That will make the code logic a bit more clear and we won't have binaries as dependencies. That sounds like a good task for @katherinelc321

Reply:
Yeah, there's definitely room for a refactor here, but I wanted to get a tool out the door as quickly as possible to help alleviate some of the issues being seen by other teams.